### PR TITLE
Honda HRV

### PR DESCRIPTION
**Pull request by cowanhm**
PR / Update for "Honda HRV 2019 Touring"
1. values.py - incl. Class, FP, FW_VERSIONS,  DBC Identical to CAR.FIT, SPEED_FACTOR 1.025
2. carstate.py - incl. carFingerprint shared with CAR.FIT, doorOpen OVERRIDE (signal unknown)
      Safety test concludes that removal of drivers seatbelt IMMEDIATELY disengages OP
3. interface.py - incl. HR-V vehicle parameters, tuning (tested for about 100 miles)
      Longitudinal tuning parameters unchanged from CAR.FIT

Thanks, Cowan Moore 330-701-3922

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -93,6 +94,9 @@ class CAR:
   CAR.FIT: [{
     57: 3, 145: 8, 228: 5, 304: 8, 342: 6, 344: 8, 380: 8, 399: 7, 401: 8, 420: 8, 422: 8, 427: 3, 428: 8, 432: 7, 464: 8, 487: 4, 490: 8, 506: 8, 597: 8, 660: 8, 661: 4, 773: 7, 777: 8, 780: 8, 800: 8, 804: 8, 808: 8, 829: 5, 862: 8, 884: 7, 892: 8, 929: 8, 985: 3, 1024: 5, 1027: 5, 1029: 8, 1036: 8, 1039: 8, 1108: 8, 1322: 5, 1361: 5, 1365: 5, 1424: 5, 1600: 5, 1601: 8
   }],
+  CAR.HRV: [{
```
Comment (vanillagorillaa) [inline selfdrive/car/honda/values.py:97]: This section can actually get removed.

Comment (cowanhm) [reply selfdrive/car/honda/values.py:97]: Yes sir, I realize that.  If it's possible to leave as a reference for other hrv'ers, as there is still some code that helps people discover their fp's.  I'm not sure the differences between HRV's at this point. 

Can we remove after further hrv users review?  Thanks

Comment (vanillagorillaa) [reply selfdrive/car/honda/values.py:97]: Actually looks like we do need it for other stuff. So leaving it is fine

**Comment by cowanhm** [comment]
Planning to conduct further testing with other commaai/openpilot branches

**Pull request closed without merging**